### PR TITLE
Export option exceptions

### LINKED ISSUE #67: If Add-ID is not selected in Export Options the export will not load
I will fix this myself.

**Describe the bug**
If Included data:  Patient identifier: Add-ID is not selected reading of an export will not work. This is correct but a sensible error message needs to be thrown [here](https://github.com/SwissClinicalTrialOrganisation/secuTrialR/blob/master/R/read_secuTrial_export.R#L71).

**To Reproduce**
The file is in the data archive of this package.
```
read_secuTrial_export("s_export_CSV-xls_CTU05_shortnames_sep_ref_no_addid.zip")
```

### LINKED ISSUE #103: If Project setup is not selected in Export Options the export will not load
**Describe the bug**
If Project setup is not specified in Export Options the export will not load. A cryptic unhelpful error is returned.

**To Reproduce**
Export without Project setup and try `read_secuTrial()`.

**Expected behavior**
Return useful Error message or allow loading.

### LINKED ISSUE #105: If Centre information is not selected in Export Options the export will not load
**Describe the bug**
If Centre information is not selected in Export Options the export will not load. A cryptic unhelpful error is returned. Probably because the centers table is missing.

**To Reproduce**
Export without Centre information and try `read_secuTrial()`.

**Expected behavior**
Return useful Error message or allow loading.

**Found by**
@nicoleb7

### PR DESCRIPTION
**Pull request by PatrickRWright**
I added a new directory with test exports to do this. NEWS.md explains what has been fixed. Basically, if the information to add the `pat_id`, `visit_name` or `centre` columns is missing they are simply not added.

One review should do but I will invite you both.

closes #67 
closes #103 
closes #105

### CODE REVIEW

**Comment by PatrickRWright** [comment]
I'm finished. Just thought of one more thing I wanted to test and added it while I was at it.

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# secuTrialR 0.6.0
+* `read_secuTrial_raw()` and `read_secuTrial()` no longer fail due to missing Add-ID, centre information or project setup in export data. Instead, adding of `pat_id` (no Add-ID), `centre` (no centre information) and `visit_name` (no project setup) to the data tables is now omitted if the accoring data for the operation is not available.
```
Comment (aghaynes) [inline NEWS.md]: accoring? relevant?

Comment (PatrickRWright) [reply NEWS.md]: yes

**Comment by aghaynes** [comment]
what does read_secutrial currently do? In my opinion, it would give warnings that the labels, etc are not applied, but still import the data. this could be done by checking if something is in the export options and then setting the 3 logicals to false if the setup stuff is not there...

**Comment by PatrickRWright** [comment]
Currently it will always try and add pat_id, centre and visit_name. This fails if the data for the procedure is missing.

I also considered adding warnings in this PR but decided not to.

I think that would be a work package related to #17. Once we have agreed on a set of suggested options (which will definitely include add-id, centre info and project setup) I suggest we tailor a verbose warning message for each option (if it differs from the suggestion) and point out what the effect is.

**Comment by aghaynes** [comment]
maybe it makes sense to skip factorize, dates and labels if project setup isn't available and give a warning (an if loop in read_secutrial would do i think)

**Pull request merged**